electronics_lib/PriceGetter.py

```diff
@@ -25,47 +26,58 @@ def run(self) -> Dict[str, int]:
 
 class GeneratePrice(BaseBackend):
     # part_number -> [ [lower-bound_1, price_1], [lower-bound_2 (and the previous upperbound), price_2], ... ]
-    PRICE_TABLE = Dict[str, List[Tuple[int, float]]] = {}
+    PRICE_TABLE: Dict[str, List[Tuple[int, float]]] = {}
 
     @classmethod
     def get_price_table(cls) -> Dict[str, List[Tuple[int, float]]]:
         if not GeneratePrice.PRICE_TABLE:
-            cur_path = os.path.dirname(__file__)
-            parts_library = os.path.relpath('resources/Pruned_JLCPCB SMT Parts Library(20220419).csv', cur_path)
+            current_path = os.getcwd()
+            parts_library = current_path + "\PolymorphicBlocks\electronics_lib\\resources\Pruned_JLCPCB SMT Parts Library(20220419).csv"
             with open(parts_library, 'r', newline='') as csv_file:
-                csv_reader = csv.DictReader(csv_file, 'r')
+                csv_reader = csv.reader(csv_file, dialect='excel')
```

Comment:
Not sure about `excel` dialect, but typically `encoding='gb2312'` works for these parts tables